Python/initconfig.c

```diff
@@ -2596,8 +2596,13 @@ config_parse_cmdline(PyConfig *config, PyWideStringList *warnoptions,
     } while (1);
 
     if (print_version) {
+#ifdef Py_GIL_DISABLED
+        printf("Python %s (free-threading)\n",
+                (print_version >= 2) ? Py_GetVersion() : PY_VERSION);
+#else
         printf("Python %s\n",
                 (print_version >= 2) ? Py_GetVersion() : PY_VERSION);
+#endif
```

Comment:
I suggest to remove parens and write the code differently:

```suggestion
        const char *version;
        version = (print_version >= 2) ? Py_GetVersion() : PY_VERSION;
#ifdef Py_GIL_DISABLED
        printf("Python %s free-threading\n", version);
#else
        printf("Python %s\n", version);
#endif
```